components/wb-bubble/wb-bubble-doc-en.html

```diff
@@ -0,0 +1,68 @@
+---
+title: Bubble - Documentation
+language: en
+category: Plugins
+description: Virtual assistant bubble that execute an action.
+altLangPage: wb-bubble-doc-fr
+dateModified: 2021-06-09
+---
+{% include alert-provisional.html siteroot="../../" %}
+<div class="wb-prettify all-pre"></div>
+
+<dl class="horizontal">
+	<dt>Status</dt>
+	<dd>Provisional</dd>
```

Comment:
Status will be "Stable", you will have to change this everywhere (including the index.json-ld file).